chimneyCats/src/test/scala/io/scalaland/chimney/cats/CatsValidatedSpec.scala

```diff
@@ -442,6 +442,52 @@ object CatsValidatedSpec extends TestSuite {
           .intoF[ValidatedNec[String, +*], long.NumScale[Int]]
           .transform ==> Validated.invalidNec("bad int")
       }
+
+      "path of error" - {
+        type V[+A] = ValidatedNec[TransformationError, A]
+
+        def printError(err: TransformationError): String =
```

Comment:
Maybe it's good candidate as `def describe: String`  on `TransformationError`?

Reply:
For abstract message type?

Reply:
Nope, doesn't make sense any more :)